phpBB/phpbb/content_visibility.php

```diff
@@ -424,6 +424,36 @@ public function set_post_visibility($visibility, $post_id, $topic_id, $forum_id,
 			'post_delete_reason'	=> truncate_string($reason, 255, 255, false),
 		);
 
+		/**
+		 * Perform actions right before the query to change post visibility
+		 *
+		 * @event core.set_post_visibility_before_sql
+		 * @var			int			visibility		Element of {ITEM_APPROVED, ITEM_DELETED, ITEM_REAPPROVE}
+		 * @var			array		post_id			Array containing all post IDs to be modified. If blank, all posts within the topic are modified.
+		 * @var			int			topic_id		Topic of the post IDs to be modified.
+		 * @var			int			forum_id		Forum ID that the topic_id resides in.
+		 * @var			int			user_id			User ID doing this action.
+		 * @var			int			timestamp		Timestamp of this action.
+		 * @var			string		reason			Reason specified by the user for this change.
+		 * @var			bool		is_starter		Are we changing the topic's starter?
+		 * @var			bool		is_latest		Are we changing the topic's latest post?
+		 * @var			array		data			The data array for this action.
+		 * @since 3.1.9-RC1
```

Comment:
Unfortunately, this didn't make it into 3.1.9. I'd very much appreciate it, if you could update this to say 3.1.10-RC1. Thank you.